spec/lucky/error_handling_spec.cr

```diff
@@ -46,6 +57,18 @@ describe Lucky::ErrorHandler do
     context.response.status_code.should eq(500)
   end
 
+  it "returns the defined status code if the raised exception defines one" do
+    error_handler = Lucky::ErrorHandler.new(action: FakeErrorAction)
+    error_handler.next = ->(_ctx : HTTP::Server::Context) {
+      raise ShortAndStoutError.new("I'm a little teapot")
```

Comment:
I had this song stuck in my head for the last hour now 😂

Reply:
LOL! Nice